_posts/2016-09-29-Rust-1.12.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: post
+title: "Announcing Rust 1.12.0"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.12.0. Rust is
+a systems programming language with the slogan "fast, safe, concurrent: pick
+three."
```

Comment:
I'm warming to Aaron's "Safe, fast, convenient"(?) especially since we're talking about some of the convenience improvements in this post.